libs/facilities/frontend/view/src/lib/components/browse/browse.page.html

```diff
@@ -1,48 +1,40 @@
 <ix-content>
-    <ix-content-header slot="header" header-title="Main Dashboard" [headerSubtitle]="subtitle">
-        <ix-button (click)="toggleView()">
-            Switch to {{ showCardList ? 'Event List' : 'Card List' }}
-        </ix-button>
-    </ix-content-header>
+	<ix-content-header [headerSubtitle]="subtitle" header-title="Main Dashboard" slot="header">
+		<ix-button (click)="toggleView()">
+			Switch to {{ showCardList ? 'Event List' : 'Card List' }}
+		</ix-button>
+	</ix-content-header>
 
-    <div *ngIf="showCardList">
-        <ix-card-list
-            label="Facilities Overview"
-            [showAllCount]="facilities.length"
-            list-style="stack"
-        >
-            @for (facility of facilities; track facility.id){
-                <ix-push-card
-                    [icon]="facility.icon"
-                    [notification]="facility.notification"
-                    [heading]="facility.heading"
-                    [subheading]="facility.subheading"
-                    [variant]="facility.variant"
-                    [routerLink]="['/facilities', facility.id]"
-                >
-                    <p>Number of water pumps: {{facility.pumps}}</p>
-                    <p>Location: {{facility.location}}</p>
-                    </ix-push-card>
-                    }
-        </ix-card-list>
-    </div>
+	@if (showCardList) { @if (facilities(); as facilities) {
+	<ix-card-list label="Facilities Overview" [showAllCount]="facilities.length" list-style="stack">
+		@for (facility of facilities; track facility.id) {
+		<ix-push-card
+			[icon]="facility.icon"
+			[notification]="facility.notification"
+			[heading]="facility.heading"
+			[subheading]="facility.subheading"
+			[variant]="facility.variant"
+			[routerLink]="['/facilities', facility.id]"
+		>
+			<p>Number of water pumps: {{ facility.pumps }}</p>
+			<p>Location: {{ facility.location }}</p>
+		</ix-push-card>
+		}
+	</ix-card-list>
+	} }
 
-    <div *ngIf="!showCardList">
-        <ix-event-list item-height="L">
-            @for (facility of facilities; track facility.id){
-                <ix-event-list-item
-                    color="color-primary"
-                    [routerLink]="['/facilities', facility.id]"
-                >
-                    <div>
-                        <strong>{{ facility.heading }}</strong>
-                        <p>{{ facility.subheading }}</p>
-                        <p>Number of water pumps: {{ facility.pumps }}</p>
-                        <p>Location: {{ facility.location }}</p>
-                    </div>
-
-                </ix-event-list-item>
-            }
-        </ix-event-list>
-    </div>
+	<div *ngIf="!showCardList">
```

Comment:
I know the problem in this file was that the way it was done before was bad. But since you tried to improve it you might as well use @else for the !showCardList. Also I am not sure about the
```
@if (facilities(); as facilities) {
```
in line 8. Right now it does not show anything for both the event list and the card list when it is empty so maybe it is unnecessary. But if it is necessary you should probably put
```
@if (facilities(); as facilities){
    @if(showCardList){
        // cardlist
    }
    @else {
        // eventlist
    }
}

```

Reply:
I appreciate your engagement! 

However this ticket should **not** improve the logic / display of items itself. I just replace the old `ngIf` since it is _deprecated_ code in angular 18 (in a way), I dont want to think about new ways of displaying sth.

Reply:
yeah I understand. I just thought since you changed the ```ngIf``` once it would make sense to replace it everywhere in that file. But it is not really part of this issue so it is okay

Reply:
I replaced it, it makes sense to do it everywhere! It was just 3 am in the morning when i contributed